eeg_models/eegnet.py

```diff
@@ -72,7 +71,9 @@ def __init__(
             nn.AvgPool2d((1, rate_drops[1])),
             nn.Dropout(dropout_rate),
             nn.Flatten(),
-            nn.Linear(f2 * (n_samples // torch.prod(rate_drops)), n_classes),
+            nn.Linear(
+                f2 * (n_samples // (rate_drops[0] * rate_drops[1])), n_classes, False
+            ),
```

Comment:
remove this changes pls. They are not needed here

Reply:
done!